packages/nextjs/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # @nhost/nextjs
 
+## 2.0.0
```

Comment:
do not merge this PR before fixing this major bump - changesets does a major bump because its peer dependency has a minor bump